libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -897,6 +988,47 @@ def Delete(self, delete_disks: bool = False) -> None:
             self.FormatLogMessage(
                 'Could not find disk: {0:s}, skipping'.format(disk_name)))
 
+  def SetTags(self, new_tags: List[str]) -> None:
+    """Sets network tags for the compute instance.
```

Comment:
Is this only for *network* tags? It seems to be generic for any kind of tags.

Reply:
tags are only used to configure firewall rules and network routes, in some reference they are referred to as network tags but not in every case. I removed network from the description and added a new description line to clarify that these are the ones used for firewall/routes management.
https://cloud.google.com/vpc/docs/add-remove-network-tags